lab_6_pipeline/pipeline.py

```diff
@@ -128,11 +257,22 @@ def convert_pos(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Extracts and converts POS from the OpenCorpora tags into the UD format
         """
+        for oc_tag, ud_tag in self._tag_mapping['POS'].items():
+            if oc_tag in tags:
+                return ud_tag
+        return 'UNKN'
```

Comment:
you should get this value from mapping, not a magic string defined in a code

Reply:
reworked